tests/stages/test_generate_viz_frames_stage.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cupy as cp
+
+import cudf
+
+# pylint: disable=morpheus-incorrect-lib-from-import
+import morpheus._lib.messages as _messages
+from morpheus._lib.messages import ResponseMemory
+from morpheus.config import Config
+from morpheus.messages import ControlMessage
+from morpheus.messages import MessageMeta
+from morpheus.messages import MultiResponseMessage
+from morpheus.stages.postprocess.generate_viz_frames_stage import GenerateVizFramesStage
+
+
+def _make_multi_response_message(df, probs):
+    df_ = df[0:len(probs)]
+    mem = ResponseMemory(count=len(df_), tensors={'probs': probs})
+
+    return MultiResponseMessage(meta=MessageMeta(df_), memory=mem)
+
+
+def _make_control_message(df, probs):
+    df_ = df[0:len(probs)]
+    cm = ControlMessage()
+    cm.payload(MessageMeta(df_))
+    cm.tensors(_messages.TensorMemory(count=len(df_), tensors={'probs': probs}))
+
+    return cm
+
+
+def test_constructor(config: Config):
+    stage = GenerateVizFramesStage(config)
+    assert stage.name == "gen_viz"
+
+    accepted_types = stage.accepted_types()
+    assert isinstance(accepted_types, tuple)
+    assert len(accepted_types) > 0
```

Comment:
A better way to test this is to verify that the two types are in `accepted_types()`

```python
accepted_union = typing.Union[accepted_types]
assert typing_utils.issubtype(ControlMessage, accepted_union)
assert typing_utils.issubtype(MultiMessage, accepted_union)
```

Reply:
Do this for all new tests added